connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala

```diff
@@ -31,6 +31,10 @@ object KustoConstants {
   val DefaultTimeoutQueueing: Int = TimeUnit.SECONDS.toMillis(40).toInt
   val MaxIngestRetryAttempts = 2
   val MaxCommandsRetryAttempts = 4
+  // val MaxStreamingBytes: Int = 4 * OneMegaByte
+  // This is just a temporary value until we actually count compressed bytes. GZIP is about 25-40% of the original size.
+  val MaxStreamingBytesUnCompressed: Int = 4 * OneMegaByte
```

Comment:
We will not count compressed bytes as the compression of Kusto is not the same as Gz.
 lets just make this configurable

Reply:
Added a new optiion : kustoStreamingIngestSize  that is 4MB by default, can be configured from the invoker of the program